api/src/main/scala/hmda/api/http/LarHttpApi.scala

```diff
@@ -66,6 +69,11 @@ trait LarHttpApi extends LarProtocol with ValidationResultProtocol {
       }
     }
 
+  def errorsAsResponse(list: List[String]): HttpResponse = {
+    val errorEntity = HttpEntity(ContentTypes.`application/json`, list.toJson.toString)
+    HttpResponse(StatusCodes.UnprocessableEntity, entity = errorEntity)
```

Comment:
Do we really want a `422`?  By [definition](https://tools.ietf.org/html/rfc4918#page-78) it seems _OK_, but it also seems WebDAV-centric, and I don't think it is a very common error code.  I think a good ol' `400` with a standard error message format (to be defined https://github.com/cfpb/hmda-platform/issues/433) is probably a more consistent option.

Maybe something like:

```json
{
  "status_code": 400,
  "errors": [
    {"code": "field-not-parsable", "'Owner Occupancy' field is not an integer"},
    {"code": "field-not-parsable", "'Date of Action' field is not an integer"}
  ]
}
```

...or...

```json
{
  "status_code": 400,
  "errors": [
    {"code": "invalid-number-of-fields", "Incorrect number of fields: 37"}
  ]
}
```

This doesn't necessarily have to be fixed in this PR.  If we wait, let's link back to this as a refactor when we do #433.